front/src/api/index.ts

```diff
@@ -36,7 +36,7 @@ type APIConfig = {
 };
 
 const DEFAULT_CONFIG: APIConfig = {
-  baseURL: "http://localhost:5000",
+  baseURL: "",
```

Comment:
Same. Let's keep the default and use environment variables or configuration files to change it.